en/docs/references/concepts/authorization/implicit-grant.md

```diff
@@ -1,8 +1,6 @@
 # Implicit Grant Type
 
-### Recommended Use
-
-The **implicit grant type** is optimized for [public clients](client-types.md) known to operate a particular redirection URI.
+The implicit grant type is optimized for [public clients](../client-types/#public-clients) known to operate a particular redirection URI.
 It is mainly used for clients that are not capable of keeping the client’s own credentials secret; for example a 'JavaScript only' application.
 Since the access token is encoded into the redirect URI, it may be exposed to other parties other than the client, including the resource owner.
```

Comment:
```suggestion
Since the access token is encoded into the redirect URI, it may be exposed to parties other than the client, including the resource owner.
```